client/src/connection/com/index.ts

```diff
@@ -118,8 +119,8 @@ export class COMSession extends Session {
       );
 
       //write an end mnemonic so that the handler knows when execution has finished
-      const codeWithEnd = `${codeWithODSPath}\n%put ${endCode};`;
-      const codeToRun = `$code=@"\n${codeWithEnd}\n"@\n`;
+      const codeWithEnd = `%put ${beginningCode};\n${codeWithODSPath}\n%put ${endCode};`;
+      const codeToRun = `$code=@'\n${codeWithEnd}\n'@\n`;
```

Comment:
A Here-string in PowerShell should start with a new line.
```suggestion
      const codeToRun = `$code=\n@'\n${codeWithEnd}\n'@\n`;
```
Then no need for extra ``` `' ```

Reply:
you are right. It works.